.github/workflows/release.yml

```diff
@@ -51,15 +51,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: echo "NEXT_VERSION=$(git cliff --bumped-version)" >> "$GITHUB_OUTPUT"
-      - name: Get release notes
```

Comment:
The change in order aims to fix the Unrelease section that shows up in the release notes:

<img width="990" alt="image" src="https://github.com/user-attachments/assets/71f69ad4-675c-4598-85b7-9ae0df2b59c9">